public/app/data/sampleDataset.json

```diff
@@ -2,579 +2,50 @@
   "data": {
     "actions": [
       {
-        "_id": "5f60873b8872f5e4125eeeb6",
-        "user": "5f60874735d84d75e4d9674d",
+        "_id": "555555328729e6001a0ee309",
+        "user": "99999b05cd601766ce72c0b9",
         "sessionId": null,
-        "data": {
-          "subpage": "5f608753ae8ec5690ad1a113"
-        },
+        "data": { "page": "111111328729e1111a0ee309" },
         "visibility": "public",
-        "space": "5f608753ae8ec5690ad1a113",
+        "space": "5ed0000003c8a33f3d5d87a5",
```

Comment:
Same thing here: the `space` id should correspond to one space in `metadata/spaceTree`.